clambcc/clambc-compiler.py

```diff
@@ -764,6 +764,24 @@ def _process_long_opt(self, rargs, values):
     def getPassthrough(self):
         return self.passthrough
 
+def printVersion():
+    compilerVersion = "Not written"
+    fileName = os.path.join("headers", "clambc-version.h")
```

Comment:
This doesn't feel right to me.  If installing clambcc to system, we wouldn't install it under `/usr/local/include/headers/clambc-version.h`, it'd be something like `/usr/local/include/clambc-version.h` or `/usr/local/include/clambc/clambc-version.h`. So `clambc-compiler.py` should try to find it in the installed path.  I'm not really sure how to do that unless we generate `clambc-compiler.py`, or generate some supporting file.  Of course, if we're doing that, why not just insert the compiler version in the generated file rather than insert the header path and then have to go parse the header.

Reply:
I just checked, and it seems that we're already part of the way there. Right now it does a "COPYONLY" on line https://github.com/Cisco-Talos/clamav-bytecode-compiler/blob/main/clambcc/CMakeLists.txt#L3

Having it configure in the version number when it does that may be as simple as removing the word "COPYONLY" on that line, and then changing the `printVersion` function to:
```py
def printVersion():
    print(f"ClamBC-Compiler @PACKAGE_VERSION@")
    print(f"ClamAV Headers @PACKAGE_VERSION@")
```